config/locales/en.yml

```diff
@@ -20,4 +20,117 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  full_title: "Open Source Friday | This Friday, invest a few hours contributing to the software you use and love."
+  Open Source Friday: "Open Source Friday"
+  index_description: "Open source is made by people just like you. This Friday, invest a few hours contributing to the software you use and love."
```

Comment:
There's some duplication between the `full_title` and `index_description`. Is there any way to share the translation between both here?